### ISSUE TITLE
Rule group header loses focus indicator while using AT

### ISSUE THREAD

**Issue by smoralesd** [microsoft/accessibility-insights-web#1568]
## Describe the bug

While using an screen reader (NVDA, JAWS), the rule group loses the focus indicator when the user activates the control (collapsing or expanding the hidden content)

**To Reproduce**

1. Go to [Accessible University before](http://www.washington.edu/accesscomputing/AU/before.html)
1. Open fast pass and go to Automated checks, wait for the results to show up
1. Enable the screen reader (NVDA or JAWS)
1. Use keyboard navigation to get to the first rule group (should be **bypass**)
1. Notice the rule group has a thin, black line as the focus indicator style
1. Activate the control (press the space or enter key)
1. Observe: the focus indicator is gone. Focus is still in the element so the user can activate the control again but the focus indicator does not comes back.

## CodePen repro example

Does not apply

## Expected behavior

Focus indicator should be present whenever the rule group button control is focused.

## Screenshots

**No AT**
![01 - no AT](https://user-images.githubusercontent.com/2837582/67704542-89808500-f972-11e9-9a6f-2c6043c89c56.gif)

**NVDA**
![02 - NVDA](https://user-images.githubusercontent.com/2837582/67704550-8daca280-f972-11e9-82b6-59fba6c1d764.gif)

**JAWS**
![03 - JAWS](https://user-images.githubusercontent.com/2837582/67704563-943b1a00-f972-11e9-810e-8cc2cec24fcf.gif)

**Narrator**
![04 - Narrator](https://user-images.githubusercontent.com/2837582/67704580-9a30fb00-f972-11e9-80f4-4be56ca03e78.gif)

## Extension (please complete the following information)

- OS: Windows 10.0.18363 Build 18363
- Version Insider 2.10.3
- Browser Microsoft Edge version 79.0.308.1
- Target Page [Accessible University before](http://www.washington.edu/accesscomputing/AU/before.html)

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

No

## Additional context

**This is already happening on prod**

Narrator looks very different as it's using it's own focus indicator with a completely different it should be a good idea to have a closer look at this case as AI-Web focus indicator may or may not be behind the Narrator focus indicator

**Comment by ferBonnin** [comment]
@waabid assigning to you since you are looking at this feature

**Comment by ferBonnin** [comment]
@iamrafan investigating this. Per conversation with him, seems to be a known Jaws issue and possibly a known NVDA issue. Please add a link to the bugs or let us know if this is something that needs to be fixed on our side :)

**Comment by iamrafan** [comment]
@ferBonnin the known JAWS issue I mentioned is related to the focus issue we are seeing on the ''more options" button in the instance card, this looks like a different one

@smoralesd this looks like an office fabric issue

Office fabric adds a class ```ms-Fabric--isFocusVisible``` to the ```<body>``` element when we tab onto one of the rule group headers
- When not using an AT, this class is not being removed if we expand the rule group header
- But when using either JAWS or NVDA, the class is being removed if we expand the rule group header. It is only added back if we tab back and forth to the header 

![image](https://user-images.githubusercontent.com/22068434/71959530-cdc8a680-31a7-11ea-86de-01b8ea9deac7.png)

**Comment by iamrafan** [comment]
We had a somewhat similar issue https://github.com/microsoft/accessibility-insights-web/issues/717 earlier, which was fixed by an Office fabric update

**Comment by ferBonnin** [comment]
@iamrafan can you check if this is fixed in Playground?

**Comment by iamrafan** [comment]
@ferBonnin I validated in Playground version 2020.1.8.2334. This issue still exists

**Comment by iamrafan** [comment]
Valdiated the fix in Insider version - 2.14.1
